ui-admin/src/forms/FormContentEditor.tsx

```diff
@@ -43,7 +44,12 @@ export const FormContentEditor = (props: FormContentEditorProps) => {
             value={editedContent}
             onChange={newContent => {
               setEditedContent(newContent)
-              onChange(true, newContent)
+              try {
+                validateFormContent(newContent)
```

Comment:
We could possibly have two different validation functions. Here, we at least know that `newContent` conforms to the FormContent type and we're validating things like non-empty question titles. Whereas in the JSON editor, the user could put in any value and the type is actually `unknown`.